src/utils/helper.js

```diff
@@ -125,75 +116,147 @@ function createDirectoryContents(
-              nodeName,
-              projectChoice,
-              isThemeProvider,
-              isMaterialUI,
-              currentDirectory
-            );
-          }
-        
       }
     }
   });
 }
 
 //to update package.json------------------------------------------------>
-function updateProjectDependencies(path, dependency) {
-  let packagefile = fs.readFileSync(`${path}/package.json`, "utf-8");
-  packagefile = JSON.parse(packagefile);
-  let newPackageFile = {
-    ...packagefile,
-    dependencies: {
-      ...packagefile.dependencies,
-      [dependency.name]: dependency.version,
-    },
-  };
-  newPackageFile = JSON.stringify(newPackageFile);
-  fs.writeFileSync(`${path}/package.json`, newPackageFile, "utf-8");
+function updateProjectDependencies(path, dependencies) {
+  dependencies.forEach((each) => {
+    const packageJsonFile = fs.readFileSync(`${path}/package.json`, "utf-8");
+    const packageJson = JSON.parse(packageJsonFile);
+    const updatedPackageJson = {
+      ...packageJson,
+      dependencies: {
+        ...packageJson.dependencies,
+        [each.name]: each.version,
+      },
+    };
+
+    const newPackageJsonFile = JSON.stringify(updatedPackageJson);
+    fs.writeFileSync(`${path}/package.json`, newPackageJsonFile, "utf-8");
+  });
 }
 
 // To update the scripts in Package.json
-function updateProjectScripts(path, script) {
-  let packagefile = fs.readFileSync(`${path}/package.json`, "utf-8");
-  packagefile = JSON.parse(packagefile);
-  let newPackageFile = {
-    ...packagefile,
-    scripts: {
-      ...packagefile.scripts,
-      [script.name]: script.command,
-    },
-  };
-  newPackageFile = JSON.stringify(newPackageFile);
-  fs.writeFileSync(`${path}/package.json`, newPackageFile, "utf-8");
+function updateProjectScripts(path, updatedscripts) {
+  updatedscripts.forEach((each) => {
+    const packageJsonFile = fs.readFileSync(`${path}/package.json`, "utf-8");
+    const packageJson = JSON.parse(packageJsonFile);
+    const updatedPackageJson = {
+      ...packageJson,
+      scripts: {
+        ...packageJson.scripts,
+        [each.name]: each.command,
+      },
+    };
+
+    const newPackageJsonFile = JSON.stringify(updatedPackageJson);
+    fs.writeFileSync(`${path}/package.json`, newPackageJsonFile, "utf-8");
+  });
+}
+
+function copyFiles(paths) {
+  paths.forEach((each) => {
+    fs.copyFileSync(each.source, each.destination, (err) => {
+      if (err) {
+        console.log("Error Found:", err);
+      }
+    });
+  });
+}
+
+function copyDirectories(paths) {
+  paths.forEach((each) => {
+    fsExtra.copy(each.source, each.destination, function (err) {
+      if (err) {
+        console.log("An error is occured");
+        return console.error(err);
+      }
+    });
+  });
+}
+
+function readFile(path) {
+  return fs.readFileSync(path, "utf8");
+}
+
+function getFilePaths(name, srcDir, destDir, backendDir) {
+  switch (name) {
+    case name === REACT_THEME_FILE_PATHS:
```

Comment:
case name === REACT_THEME_FILE_PATHS:
=>
case REACT_THEME_FILE_PATHS:

can this work?
as switch case passing name, no need to add check here